src/renderer/pages/settings-page.component.tsx

```diff
@@ -70,6 +71,7 @@ export function SettingsPage() {
   const [appVersion, setAppVersion] = useState("");
   const nav = useNavigate();
   const t = useTranslation();
+  const updaterService = AutoUpdaterService.getInstance();
```

Comment:
Cannot use the useService hook here ?

Reply:
When I tried to implement it I ended up with lots of errors, so I thought I'd do it with the expected refacto...